lib/ansible/cli/__init__.py

```diff
@@ -476,7 +476,7 @@ def pager(self, text):
                 display.display(text)
             else:
                 self.pager_pipe(text, os.environ['PAGER'])
-        elif subprocess.call('(less --version) 2> /dev/null', shell = True) == 0:
+        elif subprocess.call('(less --version) > /dev/null', shell = True) == 0:
```

Comment:
this would show any errors on screen, we probably want:

`2> /dev/null 1>&2`
or
`&> /dev/null`